community/nspr/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+mkdir build inst
+
+cd build
+CFLAGS="$CFLAGS -D_PR_POLL_AVAILABLE -D_PR_HAVE_OFF64_T -D_PR_INET6 -D_PR_HAVE_INET_NTOP -D_PR_HAVE_GETHOSTBYNAME2 -D_PR_HAVE_GETADDRINFO -D_PR_INET6_PROBE" \
```

Comment:
Are these flags necessary?

Reply:
https://git.alpinelinux.org/aports/tree/main/nspr/APKBUILD#n30

Reply:
Of course it wouldn't be smart about musl...